ext/discord-rpc-build/discord-rpc.vcxproj

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|Win32">
+      <Configuration>Debug</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|Win32">
+      <Configuration>Release</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|x64">
+      <Configuration>Debug</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|x64">
+      <Configuration>Release</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="..\discord-rpc\include\discord_register.h" />
+    <ClInclude Include="..\discord-rpc\include\discord_rpc.h" />
+    <ClInclude Include="..\discord-rpc\src\backoff.h" />
+    <ClInclude Include="..\discord-rpc\src\connection.h" />
+    <ClInclude Include="..\discord-rpc\src\msg_queue.h" />
+    <ClInclude Include="..\discord-rpc\src\rpc_connection.h" />
+    <ClInclude Include="..\discord-rpc\src\serialization.h" />
+  </ItemGroup>
+  <ItemGroup>
+    <Text Include="..\discord-rpc\src\CMakeLists.txt" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="..\discord-rpc\src\connection_win.cpp" />
+    <ClCompile Include="..\discord-rpc\src\discord_register_win.cpp" />
+    <ClCompile Include="..\discord-rpc\src\discord_rpc.cpp" />
+    <ClCompile Include="..\discord-rpc\src\dllmain.cpp" />
+    <ClCompile Include="..\discord-rpc\src\rpc_connection.cpp" />
+    <ClCompile Include="..\discord-rpc\src\serialization.cpp" />
+  </ItemGroup>
+  <PropertyGroup Label="Globals">
+    <VCProjectVersion>15.0</VCProjectVersion>
+    <ProjectGuid>{BEB0A821-3C7F-410F-A525-63AFBC69BF8F}</ProjectGuid>
+    <Keyword>Win32Proj</Keyword>
+    <RootNamespace>discordrpc</RootNamespace>
+    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
+  </PropertyGroup>
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
+    <ConfigurationType>StaticLibrary</ConfigurationType>
+    <UseDebugLibraries>true</UseDebugLibraries>
+    <PlatformToolset>v141</PlatformToolset>

```

Comment:
I think we need v140_xp or v141_xp (I don't know if the latter is a Real Thing, though)?

-[Unknown]